### ISSUE TITLE
Pansharpened images are not properly stretched 

### ISSUE THREAD

**Issue by bogitamat** [developmentseed/landsat-util#106]
Thanks for the good work and easy to use tool

I am facing a problem with pan sharpening. Pan sharpened images have reduced visual enhancement in comparison to images processed without --pansharp option (see http://tinypic.com/view.php?pic=ndsmcn&s=8#.Vd8fYmBHfuc , pan sharpened image on the left)

Would appreciate any help in the matter

**Comment by JorgeGT** [comment]
I have the same issue, on Arch Linux and Ubuntu boxes:

![See](http://i.imgur.com/ifY8sT9.png)

**Comment by gmaclennan** [comment]
This problem seems to be related to the pan sharpening calculations and the black 'nodata' around the image. When I use `--clip` as well as `--pansharpen` clipping the nodata, the color correction / histogram equalization works fine:

![lc80090612015251lgn00_bands_432_pan_clipped_thumb](https://cloud.githubusercontent.com/assets/290457/10948210/b4234ef4-82e9-11e5-9b94-cfa3d5468c50.jpg)
![lc80090612015251lgn00_bands_432_pan_thumb](https://cloud.githubusercontent.com/assets/290457/10948209/b41f650a-82e9-11e5-92a2-c3c35c74eea2.jpg)

**Comment by scisco** [comment]
@gmaclennan Great observation. Thanks for spending time figuring this out. This should help me coming up with a fix.

**Comment by gmaclennan** [comment]
Any update on this?

**Comment by gmaclennan** [comment]
Just to follow up on this, the problem seems to be with color correction and nodata, even without pansharpening. Here is a comparison of the same area, the first created with

```
landsat process landsat8image.tar.bz --clip=...
```

and the second created without `--clip` and cropping afterwards.

![clip](https://cloud.githubusercontent.com/assets/290457/11450042/7b4dddfe-9542-11e5-8e70-04db73abf639.jpg)
![noclip](https://cloud.githubusercontent.com/assets/290457/11450043/7b6e8072-9542-11e5-830a-cf633f88b6c5.jpg)

**Comment by scisco** [comment]
@gmaclennan do you have the scene numbers of the examples you mentioned above?

**Comment by gmaclennan** [comment]
Yes, the first is LC80090612015251LGN00 and the second is LC82310592014267LGN00

**Comment by danlopez00** [comment]
Same problem. I am investigating on my end as well. Scene LC81050682014217LGN00 
![screen shot 2016-03-02 at 11 48 07 am](https://cloud.githubusercontent.com/assets/727396/13484592/427a797a-e0b4-11e5-82f5-37e21ca1a033.png)
